src/patchkernel/patch_kernel_parallel.cpp

```diff
@@ -3460,18 +3470,64 @@ void PatchKernel::unsetGhostCellOwner(int id)
 void PatchKernel::clearGhostCellOwners()
 {
 	m_ghostCellOwners.clear();
+
+	setPartitioningInfoDirty(true);
+}
+
+/*!
+	Checks if the information related to the partitioning are dirty.
+
+	\result Returns true if the information related to the partitioning are
+	dirty, false otherwise.
```

Comment:
Maybe, we can inform that is a collelctive function. The same for other collective get-functions.